### PR TITLE
neonvm: revert crictl patches

### PR DESCRIPTION
**Pull request by Omrigan**
We decided to go another route, and disable outer cgroup altogether in favor of something better.

Closes https://github.com/neondatabase/cloud/issues/13266.

### CODE REVIEW

File: tests/e2e/runner-pod/01-assert.yaml [hunk 1]
```diff
@@ -13,8 +13,6 @@ status:
   conditions:
     - type: Available
       status: "True"
-    - type: Degraded
-      status: "True"
```
Comment (sharnoff) [inline tests/e2e/runner-pod/01-assert.yaml:17]: Why this change?

File: neonvm/runner/Dockerfile [hunk 2]
```diff
@@ -33,12 +22,10 @@ RUN apk add --no-cache \
     e2fsprogs \
     qemu-system-x86_64 \
     qemu-img \
-	cgroup-tools \
+    cgroup-tools \
```
[the remark this answers is not in the document]
Comment (Omrigan) [reply neonvm/runner/Dockerfile:25]: Actually, it is part of the original patch: https://github.com/neondatabase/autoscaling/pull/738/commits/5a82cd85292c91e4b17eb1704b274a52ed873de4

File: neonvm/controllers/vm_controller.go [hunk 3]
```diff
@@ -934,20 +932,7 @@ const (
...
-// The full set of outputs is:
-//
-//   - runnerUnknown, if pod.Status.Phase is Unknown
-//   - runnerPending, if pod.Status.Phase is "" or Pending
-//   - runnerRunning, if pod.Status.Phase is Running, and no containers have exited
-//   - runnerFailed, if pod.Status.Phase is Failed, or if any container has failed, or if any
-//     container other than neonvm-runner has exited
-//   - runnerSucceeded, if pod.Status.Phase is Succeeded, or if neonvm-runner has exited
-//     successfully
+// This is *similar* to the value of pod.Status.Phaseg .
```
[the remark this answers is not in the document]
Comment (Omrigan) [reply neonvm/controllers/vm_controller.go]: Done

File: tests/e2e/runner-pod/01-assert.yaml (hunk 1 above)
Comment (Omrigan) [reply tests/e2e/runner-pod/01-assert.yaml:17]: Hmm... I am not actually sure. For some reason, when I apply 3e6da9c1cac48ca082c545b64461c57ea5741139, status conditions stop updating. Any idea why it might be the case?

Comment (sharnoff) [reply tests/e2e/runner-pod/01-assert.yaml:17]: from discussion: plan is to fix our usage of status conditions in a separate PR

File: neonvm/runner/Dockerfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (sharnoff) [reply neonvm/runner/Dockerfile:25]: Oof, thanks for digging into it. No preference either way here. Ideally we should have CI check for this in the future.

**Review by sharnoff — APPROVED** [review]
Looks good. Feel free to merge when the status conditions thing is fixed
